config.js

```diff
@@ -64,9 +64,19 @@ module.exports = {
 
     // Specifies the behavior of the trace agent in the case of an uncaught exception.
     // Possible values are:
-    //   `ignore`: Take no action.
-    //   `flush`: Attempt to publish traces silence the exception.
-    //   `flushAndExit`: Attempt to publish traces and exit the process.
+    //   `ignore`: Take no action. Note that the process may termiante before all the 
+    //            traces currently buffered have been flushed to the network.
+    //   `flush`: Handle the uncaught exception and attempt to publish the traces to
```

Comment:
I thought about that too; but Silence is not accurate if there are other uncaught exception handlers.